pkg/daemon/ceph/util/util.go

```diff
@@ -53,7 +53,7 @@ func GetPortFromEndpoint(endpoint string) int32 {
 			logger.Errorf("failed to convert %q to integer. %v", portString, err)
 		}
 	}
-	//#nosec G109 -- This is just a lower bit to bigger bit conversion
+	//nolint:gosec // G109 No overflow: This is just a lower bit to bigger bit conversion
 	portInt32 := int32(port)
```

Comment:
Based on what Michael found a couple weeks ago, `Atoi()` always returns an `int` type. While we know the underlying data is supposed to be `int32`, in code we might not be certain that overflow is impossible. Instead of skipping this gosec check, I suggest we use that `ParseInt()` function to parse the string as 32 bit. That will ensure we get proper errors in the unlikely event of an issue.

Reply:
changed `Atoi` to `ParseInt`

Reply:
done, but in fact, it seems that PareInt returns an int64 type:  https://pkg.go.dev/strconv#ParseInt